mobile/src/modules/i18n/translations/id.json

```diff
@@ -0,0 +1,229 @@
+{
+  "language": "Indonesia",
+
+  "common": {
+    "album": "Album",
+    "albums": "Album",
+    "artist": "Artis",
+    "artists": "Artis",
+    "folder": "Folder",
+    "folders": "Folder",
+    "playlist": "Daftar Putar",
+    "playlists": "Daftar Putar",
+    "track": "Trek",
+    "tracks": "Trek",
+
+    "play": "Putar",
+    "prev": "Sebelumnya",
+    "next": "Selanjutnya",
```

Comment:
`Selanjutnya` seems to mean "futhermore"; I guess it could work. I'm not sure if `berikutnya` would be correct?

Reply:
In other players, "Next" is usually translated as "selanjutnya", maybe there is something wrong with the translation here, I'll check it out.